_std/defines/item.dm

```diff
@@ -212,6 +212,8 @@
 
 // for pen reagent dipping
 #define PEN_REAGENT_CAPACITY 4
+// for katana coating, applied to turf on dash. Only used for sakuride for now
+#define KATANA_REAGENT_CAPACITY 15
```

Comment:
No particular reason for this to be a global define, may as well just be on the katana type.